### ISSUE TITLE
Doesn't work after updating config.php

### ISSUE THREAD

**Issue by AlexUrrutia** [digitalfotografen/AWS-rekognition-demo#2]
Hello the script always returns Access Denied, it seems the config.php does not make any effect, it keeps trying to connect to ulrik S3 bucket

`php src/AwsTest.php -v                                    
PHP Fatal error:  Uncaught exception 'Aws\S3\Exception\S3Exception' with message 'Error executing "List
Objects" on "https://s3-eu-west-1.amazonaws.com/ulrik.ekognition.test?encoding-type=url"; AWS HTTP erro
r: Client error: `GET https://s3-eu-west-1.amazonaws.com/ulrik.ekognition.test?encoding-type=url` resul
ted in a `403 Forbidden` response:                                                                     
<?xml version="1.0" encoding="UTF-8"?>                                                                 
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>1547BC (truncated...)       
 AccessDenied (client): Access Denied - <?xml version="1.0" encoding="UTF-8"?>                         
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>1547BC8875D3720C</RequestId>
<HostId>Zw0dXv+z4AmQhmuaFfx5q+Cy+QrEXPbWgr6iFbBwQ6XSJDmqsaxHq75rW78086JulTrvN5hUnO4=</HostId></Error>' 
                                                                                                       
exception 'GuzzleHttp\Exception\ClientException' with message 'Client error: `GET https://s3-eu-west-1.
amazonaws.com/ulrik.ekognition.test?encoding-type=url` resulted in a `403 Forbidden` response:         
<?xml version="1.0" encoding="UTF-8"?>                                                                 
<Error><Code>AccessDenied</Code><Messag in /var/www/reko2/rekognition/vendor/aws/aws-sdk-php/src/Wrappe
dHttpHandler.php on line 192     `

**Comment by digitalfotografen** [comment]
Hi,
I am sorry but I can't reproduce this error. Are You using the last version from the master branch?

The "ulrik.ekognition.test" was hardcoded in a very early version (I admit, that was really stupid of me). So please make sure You use the latest master version.

Let me know if the problem persists.
Ulrik

**Comment by AlexUrrutia** [comment]
Yes I am using the master and it seems the URL is still hardcoded somewhere...
